packages/api/jest.config.cjs

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  testEnvironment: "node",
+  testMatch: ["**/?(*.)+(spec|test).ts?(x)"],
+  testPathIgnorePatterns: ["dist", "node_modules"],
+  transformIgnorePatterns: [],
+  transform: {
+    "\\.[jt]s$": ["babel-jest", { cwd: __dirname }],
+  },
+  // Jest struggles to read these mappings from the multiformats packages, which seem to be specifying them just fine as far as I can tell.
+  moduleNameMapper: {
```

Comment:
:roll_eyes: Such a shame we have to repeat this in all our packages' configs.

Reply:
Is there any particular reason why we can't set this up at the packages root? Is the difference in babel configs the blocker?

Reply:
Maybe we can! I'm not sure how configuration "inheritance" works for Jest.